shaded-deps-parent/pom.xml

```diff
@@ -145,6 +145,7 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
       <plugins>
         <plugin>
           <groupId>org.sonatype.plugins</groupId>
```

Comment:
Same comment as above re: excluding the shaded-deps-xxxx modules from Maven Central.